hack/test.sh

```
@@ -19,19 +20,31 @@ if [ $HOME == "/" ]; then
-# Capture the test result to exit on error after coverage.
+# Build outside --timeout, then run the precompiled binaries.
+echo ${GINKGO} build ${GINKGO_BUILD_ARGS} "<omitted>"
+GOGC=50 ${GINKGO} build ${GINKGO_BUILD_ARGS} ${TEST_PACKAGES} || exit $?
+
+TEST_BINARIES=()
+for pkg in ${TEST_PACKAGES}; do
+  bin="${pkg}/$(basename "${pkg}").test"
+  [ -x "${bin}" ] && TEST_BINARIES+=("${bin}")
+done
+
+echo ${GINKGO} run ${GINKGO_ARGS} ${GINKGO_EXTRA_ARGS} "<omitted>"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)